liblouis/compileTranslationTable.c

```diff
@@ -1406,12 +1406,14 @@ _lou_extParseChars(const char *inString, widechar *outString) {
 	   For this function, the functional nature should be maintained,
 	   i.e. the same input -> the same behavior.
 	   Therefore, `errorCount` should not be externally influenced. */
+	int oldErrorCount = errorCount;
 	errorCount = 0;
 	parseChars(NULL, &result, &wideIn);
 	if (errorCount) {
 		errorCount = 0;
 		return 0;
 	}
+	errorCount = oldErrorCount;
```

Comment:
See [my other comment](https://github.com/liblouis/liblouis/pull/1590#discussion_r1723990611). By fixing `parseChars` (the return value) we would avoid this ugly code.